docs/source/intro/datasets.rst

```diff
@@ -44,47 +44,64 @@ For tabular data files, the software accepts a set of predetermined column names
++----------+---------------------------------------------------------------------------------------------------------+-----------+
+| Priors_1 | final_included, label, label_included, included_label, included_final, included, included_flag, include | no        |
++----------+---------------------------------------------------------------------------------------------------------+-----------+
+| Priors_2 | debug_label                                                                                             | no        |
 +----------+---------------------------------------------------------------------------------------------------------+-----------+
 
-\* Either a title or an abstract is mandatory.
-
-
-Metadata
---------
-
-Each entry in the dataset should hold metadata on a paper. Mandatory metadata
-are ``title`` or ``abstract``. If both title and abstract are available the
-text is combined and used for training the model. If ``keywords`` and/or
-``author`` are available it can be used for searching prior knowledge. Note
-the information is not shown during the screening phase and is also not used
-for training the model, but the information is available via the API. If
-``DOI`` is available it will be displayed during the screening phase as a
-clickable hyperlink to the full text document. Note by using ASReview you do
-not automatically have access to full-text and if you do not have access you
-might want to read this `blog post <https://asreview.nl/tools-that-work-well-with-asreview-google-scholar-button/>`__.
-
-When using the :doc:`ASReview command line interface for simulation
-<../API/cli/>`, an additional binary variable to indicate labeling decisions
-(``0`` = irrelevant, ``1`` = relevant) is required for ALL records. In
-ASReview LAB, if labels are available for a part of the dataset (i.e., :doc:`partly
-labeled data <../features/pre_screening>`), the labels will be automatically detected and used for prior
-knowledge.
-
-You can explore a previously labeled dataset in ASReview LAB by adding an
-extra column called ‘debug_label’, indicating the relevant and irrelevant
-records with ones and zeroes. The relevant records will show up green during
-screening.
+\* Only a title or an abstract is mandatory.
+
+**ID**   If your data contains a column titled ``record_id`` it needs to
+consists only of intigers, and it should contain no missing data and no
+duplicates, otherwise you will receive an error. If there is no ``record_id``
+it will be automtically generated by the software. This column can also be
+used for the Simulation Mode to select prior knowledge.
+
+**Title - Abstract**
+Each record (i.e., entry in the dataset) should hold metadata on a paper.
+Mandatory metadata are only ``title`` or ``abstract``. If both title and
+abstract are available the text is combined and used for training the model.
```

Comment:
This line was in the previous version of this document as well, but I think it is a bit unclear. 

I thought that if you have at least an title **or** an astract, the text will be used for training. (right?)

Then, it would be clearer to say: 

> Mandatory metadata are only ``title`` or ``abstract``. If both title and
> abstract are available the text is combined and used for training the model. If only one of the two is available, it's text will be used for training the model as is.